_docs/quickstarts/explain/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: IAM Explain
+title: Explain
```

Comment:
Felix M had indicated to me that the "official" product name would be IAM Explain; can we confirm?